examples/with-next-multilingual/README.md

```diff
@@ -0,0 +1,7 @@
+# with-next-multilingual - internationalization using next-multilingual
+
+The `next-multilingual` module is a unique i18n module for Next.js that offers localized URLs and modular string management on top of other standard features required for multilingual applications.
```

Comment:
Hey! Could you add the repository here instead of linking out? Otherwise, `create-next-app -e with-next-multilingual` won't work.

Reply:
Do you mean adding the entire https://github.com/Avansai/next-multilingual-example repo? Does this mean I will have to keep it in sync?

Reply:
We'd prefer that, but I won't block this for it.

Reply:
Thanks - let me see if I can find a way to semi-automate that into our pipeline and maybe do a follow-up PR.